### PR TITLE
parallelize card clearing and tracker generation

### PR DESCRIPTION
**Pull request by benadida**
Deleting the card takes ~3-4s. Generating the tracker takes 4s. Let's put them in parallel and save ourselves 4s of printing dialogs.

Also, handle error conditions of no tracker generated and card not properly cleared.

### CODE REVIEW

File: src/pages/PrintOnlyScreen.tsx [hunk 1]
```diff
@@ -86,54 +97,89 @@ const PrintOnlyScreen = ({
...
           if (useElectionGuard) {
-            // no delay in next state because that state takes a while
-            setPrintingState('GENERATE_TRACKER')
-          } else {
-            printBallotTimer.current = window.setTimeout(() => {
-              setPrintingState('PRINT_DONE')
-            }, printingMessageTimeoutSeconds * 1000)
+            // don't await, just kick off both jobs
+            generateTrackingCode()
+            markCardUsed()
```
Comment (eventualbuddha) [inline src/pages/PrintOnlyScreen.tsx]: Seems like these should be `await`ed and the state change should happen after this. Something like this:

```ts
await Promise.all([generateTrackingCode(), markCardUsed()])
setPrintingState(…)
```

Comment (benadida) [reply src/pages/PrintOnlyScreen.tsx]: I think that's an alternative yes, but if you see how I handled the state and the error conditions for each one, I think my way is just as good and possibly simpler for error conditions?

Comment (eventualbuddha) [reply src/pages/PrintOnlyScreen.tsx]: I'm just concerned that nothing is handling the possible exceptions from those functions, but then again mine doesn't really either. In general I dislike seeing a promise-returning function that isn't `await`ed, but I think yours is fine for now.

Comment (benadida) [reply src/pages/PrintOnlyScreen.tsx]: that's fair, I can see how I'm abusing the async / await syntax a bit here. Let's discuss a better way! I'll merge into the branch and file a ticket.

Comment (eventualbuddha) [reply src/pages/PrintOnlyScreen.tsx]: My first impression: wrap calls in `try/catch` and do a `setError` or something in `catch`.

Comment (benadida) [reply src/pages/PrintOnlyScreen.tsx]: see #1166

**Pull request merged**